external/boost/bimap/unordered_set_of.hpp

```diff
@@ -0,0 +1,27 @@
+// Copyright 2022 D-Wave Systems Inc.
+//
+//    Licensed under the Apache License, Version 2.0 (the "License");
+//    you may not use this file except in compliance with the License.
+//    You may obtain a copy of the License at
+//
+//        http://www.apache.org/licenses/LICENSE-2.0
+//
+//    Unless required by applicable law or agreed to in writing, software
+//    distributed under the License is distributed on an "AS IS" BASIS,
+//    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+//    See the License for the specific language governing permissions and
+//    limitations under the License.
+//
+// Excerpted from and/or inspired by implementations found in the Boost Library
```

Comment:
I think we might want to put this at the top, rather than below the header. But I would check with @silviacorbellaro if you haven't already.

Reply:
There is no strict requirement to have it at he top, as long as it is prominently displayed. It looks okay to me.